### DISCUSSION TITLE
Downgrade caused a fatal error for SampleValetDriver::serves

### DISCUSSION THREAD

**Discussion by cbloss** [laravel/valet#1399] [General]
Hey!! I upgraded to Valet 4.0 and was experiencing horrible lag time for some odd reason. I tried to downgrade to 3.2.1. I had to remove global composer vendor file and lock file but eventually got there. Now I'm getting this fatal error on all the sites: 

Fatal error: Declaration of Valet\Drivers\Custom\SampleValetDriver::serves(string $sitePath, string $siteName, string $uri): bool must be compatible with Valet\Drivers\ValetDriver::serves($sitePath, $siteName, $uri) in /Users/amanda/.config/valet/Drivers/SampleValetDriver.php on line 12

I tried unlinking and re-linking one site and still get the same error. What on earth can cause this? They are all using PHP 8.1

Help?

**Comment by drbyte** [comment]
Probably a v4 file (with different syntax) that didn't get removed/overwritten with the v3 downgrade.

I think you can probably fix it by deleting the file using this shell command:

```sh
rm ~/.config/valet/Drivers/SampleValetDriver.php
```

That particular file is only an example and serves no functional purpose unless customized, so is safe to delete. Subsequent re-installs of Valet will probably recreate the file anyway.

**Reply by cbloss** [reply]
That did it!! Thanks!!
